helm/temporal-worker-controller/templates/_helpers.tpl

```diff
@@ -0,0 +1,50 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "temporal-worker-controller.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
```

Comment:
Nit: I prefer just using `$.Chart.Name` in templates. There's no real need for overriding the chart name and that also avoids the complexity of truncating custom name overrides to fit k8s name limits.
```suggestion
```